dataactbroker/handlers/fileHandler.py

```diff
@@ -1557,12 +1562,14 @@ def list_submissions(page, limit, certified, sort='modified', order='desc', is_f
     query = query.order_by(sort_order)
 
     total_submissions = query.count()
+    min_last_mod = min_mod_query.one()[0]
```

Comment:
While there's basically no chance this could return zero, could this use `first()` or `one_or_none()`
and the response include:
`'min_last_modified': str(min_last_mod[0]) if min_last_mod else None`.
Just a personal preference, you're welcome to ignore.

Reply:
Disregarding, `min()` returns `None` even if there are no results.